src/main/webapp/app/exercises/modeling/assess/modeling-assessment-editor/modeling-assessment-dashboard.component.html

```diff
@@ -5,12 +5,26 @@ <h4>
     <jhi-assessment-filters (filterChange)="updateFilteredSubmissions($event)" [submissions]="submissions"></jhi-assessment-filters>
     <jhi-assessment-warning [exercise]="exercise" [submissions]="filteredSubmissions"></jhi-assessment-warning>
     <h4>
-        <small [ngStyle]="assessedSubmissions === submissions.length && submissions.length > 0 ? { color: '#5cb85c' } : {}">
-            <strong>Progress: {{ (assessedSubmissions / submissions.length) * 100 | number: '0.0-0' }} % </strong>({{ assessedSubmissions }} out of
-            {{ submissions.length }} submitted models assessed)
-        </small>
+        <ng-container *ngIf="submissions && submissions.length === 0">
+            <small [ngClass]="'text-warning'">
```

Comment:
```suggestion
            <small class="text-warning">
```

Reply:
Thanks! Solved in 2583cef5355323aad2b40177f34b236a4a54bf3d